File: python/8 kyu/382.py
# https://www.codewars.com/kata/573f5c61e7752709df0005d2/train/python

# Passed

def merge_arrays(first, second): 
    result = list(set(first + second))
    result.sort()
    return result

output = merge_arrays([1, 3, 5], [2, 4, 6])
print(output)

File: nodejs/7 kyu/382.js
// https://www.codewars.com/kata/56b7f2f3f18876033f000307/train/javascript

// Passed

function inAscOrder(arr) {
  let result = arr.every(
    (elem, idx) => idx === arr.length - 1 || elem <= arr[idx + 1],
  );
  return result;
}

const output = inAscOrder([1, 2, 4, 7, 19]);
console.log(output);